### ISSUE TITLE
Fire Red Glitches after applying "AnyGame_ExpandedOWs.hma" script

### ISSUE THREAD

**Issue by XiscoLeith** [haven1433/HexManiacAdvance#224]
Hi! I have an issue where if I apply the AnyGame_ExpandedOWs.hma and open my Fire Red ROM, the ROM glitches. Here are the details.

-The ROM does not have a .sav file attached to it: I'm testing it as if I was someone who just downloaded the ROM.
-The ROM opens just fine and plays just fine; I can even select my gender with no issues. The ROM only glitches once Oak asks me what my name is: right before the prompt to choose a name shows up, the game glitches and the ROM suddenly resets.

Any idea what could be causing this issue? I can provid any more data if you guys need to.

**Comment by haven1433** [comment]
@XiscoLeith sorry for taking so long to get back to you on this.

So I just used the script on a vanilla (unedited) copy of FireRed, and started the game... and it seemed to work fine. I was able to get through the intro with no issues, with my character now walking around in my room.

Are you using the most recent version of HMA? I did this testing with 0.5.6.1, downloaded from the Discord releases channel.
